installtest fails on pypy3 with latest package levels

From Travis run on pypy3 with latest package levels (see https://travis-ci.org/github/pywbem/nocaselist/jobs/709856857):
```
$ make installtest
Makefile: Running install tests
tests/installtest/test_install.sh dist/nocaselist-1.0.0.dev1-py2.py3-none-any.whl dist/nocaselist-1.0.0.dev1.tar.gz python
Preparing for the tests
Creating test directory: tests/installtest/../../tmp_installtest
Creating source archive unpack directory: tests/installtest/../../tmp_installtest/src_dist
Removing .egg file: tests/installtest/../../dist/nocaselist*.egg
Testcase test1: Pip install from repo root directory: tests/installtest/../..
Saving location of current virtualenv: /home/travis/virtualenv/pypy3.6-7.1.1
Before creating virtualenv: tests/installtest/../../tmp_installtest/virtualenvs/nocaselist_test_test1
Python version: Python 3.6.1 (784b254d6699, Apr 14 2019, 10:22:42)
[PyPy 7.1.1-beta0 with GCC 6.2.0 20160901] from /home/travis/virtualenv/pypy3.6-7.1.1/bin/python
Pip version: pip 20.1.1 from /home/travis/virtualenv/pypy3.6-7.1.1/site-packages/pip (python 3.6)
Creating virtualenv: tests/installtest/../../tmp_installtest/virtualenvs/nocaselist_test_test1
Activating virtualenv: tests/installtest/../../tmp_installtest/virtualenvs/nocaselist_test_test1
Error: Command failed with rc=1: source tests/installtest/../../tmp_installtest/virtualenvs/nocaselist_test_test1/bin/activate, output follows:
tests/installtest/test_install.sh: line 230: tests/installtest/../../tmp_installtest/virtualenvs/nocaselist_test_test1/bin/activate: No such file or directory
Makefile:573: recipe for target 'installtest' failed
make: *** [installtest] Error 1
The command "make installtest" exited with 2.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -109,14 +109,13 @@ jobs:
 #      env:
 #        - PACKAGE_LEVEL=latest
 
-# TODO: pypy3 disabled due to install error with typed-ast (from pylint)
 #    - os: linux
 #      language: python
 #      python: "pypy3"  # currently Python 3.6.1, PyPy 7.1.1-beta0
 #      env:
 #        - PACKAGE_LEVEL=minimum
 
-# TODO: pypy3 disabled due to install error with typed-ast (from pylint)
+# TODO: pypy3 with latest package levels disabled due to virtualenv creation issue
 #    - os: linux
 #      language: python
 #      python: "pypy3"  # currently Python 3.6.1, PyPy 7.1.1-beta0

--- a/tests/installtest/test_install.sh
+++ b/tests/installtest/test_install.sh
@@ -11,23 +11,25 @@ PACKAGE_NAME=nocaselist
 
 function abspath()
 {
-    # generate absolute path from relative path
-    # $1     : relative filename
-    # return : absolute path
-    if [ -d "$1" ]; then
-        # dir
-        (cd "$1"; pwd)
-    elif [ -f "$1" ]; then
-        # file
-        if [[ $1 == */* ]]; then
-            echo "$(cd "${1%/*}"; pwd)/${1##*/}"
-        else
-            echo "$(pwd)/$1"
-        fi
-    else
-        error "Path does not exist: $1"
-        exit 1
-    fi
+    # return absolute path name, normalizing ".."
+    # $1     : input path name
+    # return : normalized absolute path name
+    python -c "import os; print(os.path.abspath('$1'))"
+
+    # if [ -d "$1" ]; then
+    #     # dir
+    #     (cd "$1"; pwd)
+    # elif [ -f "$1" ]; then
+    #     # file
+    #     if [[ $1 == */* ]]; then
+    #         echo "$(cd "${1%/*}"; pwd)/${1##*/}"
+    #     else
+    #         echo "$(pwd)/$1"
+    #     fi
+    # else
+    #     error "Path does not exist: $1"
+    #     exit 1
+    # fi
 }
 
 MYNAME=$(basename "$0")
@@ -35,7 +37,7 @@ MYDIR=$(dirname "$0")    # Directory of this script, as seen by caller
 
 # Repo root dir, as seen by caller
 # Using MYDIR makes the script run with any caller's CWD.
-ROOT_DIR="$MYDIR/../.."
+ROOT_DIR=$(abspath "$MYDIR/../..")
 
 # Detect and default PACKAGE_LEVEL (minimum/latest)
 if [[ -z $PACKAGE_LEVEL ]]; then
@@ -153,6 +155,10 @@ function make_virtualenv()
     virtualenv_debug_opts=""
   fi
   run "virtualenv -p $python_cmd_path $virtualenv_debug_opts $envdir" "Creating virtualenv: $envdir"
+  if [[ "$DEBUG" == "true" ]]; then
+    echo "Debug: Listing files in $envdir/bin"
+    ls -al $envdir/bin
+  fi
   run "source $envdir/bin/activate" "Activating virtualenv: $envdir"
 
   verbose "Virtualenv before reinstalling base packages:"

